OlsztynHooligans/src/app/login/login.component.ts

```diff
@@ -55,7 +55,8 @@ export class LoginComponent implements OnInit {
 
   onSubmit() {
     this.dataService.postusers(this.user1).subscribe(data => {
-      this.router.navigate([`/dashboard`]);
+      this.router.navigate([`/dashboard`]),
+      this.store.dispatch(new LoginActions.Login(this.logged));
```

Comment:
Wywal tutaj tego router,
jak masz dispatch new LoginAction to tam w effecie jak success to robisz this router navigate